src/controller/python/chip/clusters/command.cpp

```diff
@@ -181,6 +237,83 @@ PyChipError pychip_CommandSender_SendCommand(void * appContext, DeviceProxy * de
     return ToPyChipError(err);
 }
 
+PyChipError pychip_CommandSender_SendBatchCommands(void * appContext, DeviceProxy * device, uint16_t timedRequestTimeoutMs,
+                                                   uint16_t interactionTimeoutMs, uint16_t busyWaitMs, bool suppressResponse,
+                                                   size_t n, ...)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+    VerifyOrReturnError(device->GetSecureSession().HasValue(), ToPyChipError(CHIP_ERROR_MISSING_SECURE_SESSION));
+    auto remoteSessionParameters = device->GetSecureSession().Value()->GetRemoteSessionParameters();
+    CommandSender::ConfigParameters config;
+    config.SetRemoteMaxPathsPerInvoke(remoteSessionParameters.GetMaxPathsPerInvoke());
+
+    std::unique_ptr<CommandSenderCallback> callback =
+        std::make_unique<CommandSenderCallback>(appContext, /* isBatchedCommands =*/true);
```

Comment:
In what sense. Everything is contained within this file, if we want to do that at a future time, otherwise I think you end up with more lines of code for duplicating certain calles